src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -64,7 +64,7 @@ public void Ctor_ExpectedDefaultValues_NotUapPlatform()
 
         [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "UAP won't send requests through a custom proxy")]
         [OuterLoop] // TODO: Issue #11345
-        [Fact]
```

Comment:
Nit: Add comment with bug #?

Reply:
Good idea, will re-trigger tests but worth it.